app/presenters/summary/components/saving.rb

```diff
@@ -26,7 +27,7 @@ def answers # rubocop:disable Metrics/MethodLength, Metrics/AbcSize
             :are_wages_paid_into_account, saving.are_wages_paid_into_account
           ),
           Components::ValueAnswer.new(
-            :account_holder, saving.account_holder
+            :saving_ownership_type, saving.ownership_type
```

Comment:
If I'm understanding them correctly, designs are saying to leave this row out if partner == no

Reply:
Yes, I'm going to talk to design about this post hoc. I'm planning to extract the frontend side of ownership stuff in the National Savings Certificate feature so will address this then.